cli/cmds.go

```diff
@@ -624,3 +637,19 @@ func upgrade012File(fs afero.Fs, log *logrus.Logger, filename string, fmtverbs,
 
 	return &br, err
 }
+
+func reindent(formatted string, original string) (reindented string) {
```

Comment:
I am not familiar with RST files so could you add some comments explaining how this works for them? thanks

Reply:
No problem! These [rst blocks](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks) are in effect as long as we have indentation and will be ended once leading indentation is at the outside level again. The formatter removes leading indentation per default so this function was intended to restore leading indentation again!

Reply:
This may also apply to markdown as per #51

Reply:
I wonder if we should just be smart and detect how indented that first line is and use that indentation by default instead of default to zero as it affects both file formats?

Reply:
That simplifies the code a bit; i like it. I'll change the logic to reindent always.

Reply:
> That simplifies the code a bit; i like it. I'll change the logic to reindent always.

ah, we need the reindent switch ( for go we want to strip leading whitespace ). Ill return "true" for markdown though.

Reply:
Lets address #51 in a subsequent PR i'd say